src/pages/edit_resource/description_step/TagsField.tsx

```diff
@@ -73,51 +101,73 @@ const TagsField: FC<TagsFieldProps> = ({ setAllChangesSaved }) => {
+  };
+
   return (
     <StyledSchemaPartColored color={Colors.DescriptionPageGradientColor3}>
       <StyledContentWrapper>
         <Field name={'tags'}>
           {({ field }: FieldProps) => (
             <Grid container alignItems="center" spacing={2}>
               <Grid item xs={10}>
-                <StyledAutoComplete
-                  {...field}
-                  freeSolo
-                  multiple
-                  options={[]}
-                  onChange={(_: ChangeEvent<unknown>, value: string[]) => {
-                    saveTagsChanging(field.name, value);
-                  }}
-                  renderTags={(value: any, getTagProps: any) =>
-                    value.map((option: any, index: number) => (
-                      <StyledChip
-                        deleteIcon={<StyledCancelIcon />}
-                        data-testid={`tag-chip-${index}`}
-                        label={option}
-                        {...getTagProps({ index })}
+                <StyledAutoCompleteWrapper>
+                  <Autocomplete
+                    {...field}
+                    freeSolo
+                    multiple
+                    id="register-tags-input"
+                    noOptionsText={t('common.no_options')}
+                    options={options}
+                    loading={loading}
+                    filterSelectedOptions
+                    getOptionSelected={(option: string, value: string) => option.toLowerCase() === value.toLowerCase()}
+                    onChange={(_: ChangeEvent<unknown>, valueArray: string[]) => {
+                      saveTagsChanging(field.name, valueArray);
+                    }}
+                    renderOption={(option: string) => <span data-testid={'tag-option'}>{option}</span>}
+                    renderTags={(value: any, getTagProps: any) =>
+                      value.map((option: any, index: number) => (
+                        <StyledChip
+                          deleteIcon={<StyledCancelIcon />}
+                          data-testid={`tag-chip-${index}`}
+                          label={option}
+                          {...getTagProps({ index })}
+                        />
+                      ))
+                    }
+                    renderInput={(params: AutocompleteRenderInputParams) => (
+                      <TextField
+                        {...params}
+                        id="resource-feature-tags"
+                        label={t('resource.metadata.tags')}
+                        helperText={t('resource.add_tags')}
+                        variant="filled"
+                        onChange={handleChange}
+                        fullWidth
+                        data-testid="resource-tags-input"
+                        onBlur={(event) => {
+                          const value = event.target.value;
+                          const tags = value
+                            .split(/[|,;]+/)
```

Comment:
Ser at regexen min fra tidligere ikke fungerer som den skal. Har glemt å legge til den flere plasser. (sikkert jeg som har glemt å implementere ordentlig i første versjon). 
Skriver man "dlr, læring" og så trykker tab, så fungerer regexen.
Skriver man "dlr, læring" og så trykker "enter" så får man ikke regex. 
Foreslår at vi gjør det enkelt å kaster ut regexen? evt. legge til den på onChange.

Reply:
jeg gjorde et forsøk, men det ble ganske kompleks kode om vi skal støtte både tab og splitting på komma o.l.   Jeg foreslår å gå for det som støttes av material-ui-komponenten. Så får vi heller endre om det kommer sterke ønsker fra bruker-testene

Reply:
onBlur funksjonen må nesten være med for at beskrivelsen til stine skal fungere, men regexen " .split(/[|,;]+/)" trenger ikke å være med.

Reply:
jeg endret teksten også, ja

Reply:
Ahh, det så jeg ikke. Da godkjenner jeg